### PR TITLE
Servicebus integration test - just getting started.

### PR DESCRIPTION
**Pull request by noodlefrenzy**
Encoded creds in .travis.yml, initial send/receive test for ServiceBus queues.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,39 +1,28 @@
 language: node_js
-
 node_js:
-  - "0.10"
-  - "0.12"
-
+- '0.10'
```
Comment (mbroadst) [inline .travis.yml]: a lot of these indentations seem incorrect

File: lib/types/message.js [hunk 2]
```diff
@@ -379,7 +379,13 @@ Message.prototype.encode = function(codec, buf) {
   if (this.properties) codec.encode(this.properties, buf);
   if (this.applicationProperties) codec.encode(this.applicationProperties, buf);
   if (this.body.length > 1) {
-    codec.encode(new AMQPSequence(this.body), buf);
+    if (this.body instanceof Buffer) {
```
Comment (mbroadst) [inline lib/types/message.js]: whoops, my bad :)

File: test/integration/servicebus/config.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+module.exports = {
+
```
Comment (mbroadst) [inline test/integration/servicebus/config.js]: we should investigate a way to unify the integration test config files, so like protocol, host, address/target, username, password are all sort of generic? (I'm saying just a future thought, not for this PR).

The test-fixture class could eventually provide an easy means for combining these into the "config.address" variable, etc.

**Comment by mbroadst** [comment]
only one actual comment, otherwise LGTM

File: .travis.yml (hunk 1 above)
Comment (noodlefrenzy) [inline .travis.yml]: Fixed. I was using travis commands to add the secure keys, and it mucked with the formatting.

File: test/integration/servicebus/config.js (hunk 3 above)
Comment (noodlefrenzy) [inline test/integration/servicebus/config.js]: I agree, ish. I think address makes sense, and defaultLink/queueName/whatever makes sense, but beyond that I'm less sure. I'm hoping to write some tests for topics this week - those are different enough to muck up any assumptions about commonalities, I'm hoping.

File: .travis.yml [hunk 4]
```diff
@@ -12,15 +11,6 @@ branches:
   except:
     - gh-pages
 
-# addons:
```
Comment (mbroadst) [inline .travis.yml]: not to be too nitpicky, but these are useful comments for me considering Travis is currently in the final stages of approving my request to whitelist the qpid-testing apt-source

**Pull request merged**